.sql/data.sql

```diff
@@ -0,0 +1,246 @@
+-- phpMyAdmin SQL Dump
+-- version 4.6.6deb1+deb.cihar.com~xenial.2
+-- https://www.phpmyadmin.net/
+--
+-- Host: localhost
+-- Czas generowania: 24 Paź 2018, 00:34
+-- Wersja serwera: 5.7.20-0ubuntu0.16.04.1
+-- Wersja PHP: 7.2.11-2+ubuntu16.04.1+deb.sury.org+1
+
+SET SQL_MODE = "NO_AUTO_VALUE_ON_ZERO";
+SET time_zone = "+00:00";
+
+--
+-- Baza danych: `adfind`
+--
+
+-- --------------------------------------------------------
+
+--
+-- Struktura tabeli dla tabeli `Category`
+--
+
+CREATE TABLE `Category` (
+  `id` int(11) NOT NULL,
+  `name` varchar(255) NOT NULL
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+--
+-- Zrzut danych tabeli `Category`
+--
+
+INSERT INTO `Category` (`id`, `name`) VALUES
+(1, 'Photos'),
+(2, 'Pictograms'),
+(3, 'Illustrations'),
+(4, 'Documents');
+
+-- --------------------------------------------------------
+
+--
+-- Struktura tabeli dla tabeli `File`
+--
+
+CREATE TABLE `File` (
+  `id` int(11) NOT NULL,
+  `name_add` varchar(255) NOT NULL,
+  `name_view` varchar(255) DEFAULT NULL,
+  `name_hash` varchar(255) NOT NULL,
+  `subcategory_id` int(11) DEFAULT NULL
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+
+-- --------------------------------------------------------
+
+--
+-- Struktura tabeli dla tabeli `Format`
+--
+
+CREATE TABLE `Format` (
+  `id` int(11) NOT NULL,
+  `name` varchar(255) NOT NULL
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+--
+-- Zrzut danych tabeli `Format`
+--
+
+INSERT INTO `Format` (`id`, `name`) VALUES
+(1, 'jpeg'),
+(2, 'png'),
+(3, 'svg'),
+(4, 'eps'),
+(5, 'txt'),
+(6, 'doc'),
+(7, 'ppt'),
+(8, 'xls');
+
+-- --------------------------------------------------------
+
+--
+-- Struktura tabeli dla tabeli `MapFormat`
+--
+
+CREATE TABLE `MapFormat` (
+  `id` int(11) NOT NULL,
+  `file_id` int(11) DEFAULT NULL,
+  `format_id` int(11) DEFAULT NULL
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+
+-- --------------------------------------------------------
+
+--
+-- Struktura tabeli dla tabeli `MapTag`
+--
+
+CREATE TABLE `MapTag` (
+  `id` int(11) NOT NULL,
+  `file_id` int(11) DEFAULT NULL,
+  `tag_id` int(11) DEFAULT NULL
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+-- --------------------------------------------------------
+
+--
+-- Struktura tabeli dla tabeli `Subcategory`
+--
+
+CREATE TABLE `Subcategory` (
+  `id` int(11) NOT NULL,
+  `name` varchar(255) NOT NULL,
+  `category_id` int(11) DEFAULT NULL
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+-- --------------------------------------------------------
+
+--
+-- Struktura tabeli dla tabeli `Tag`
+--
+
+CREATE TABLE `Tag` (
+  `id` int(11) NOT NULL,
+  `name` varchar(255) NOT NULL
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+--
+-- Indeksy dla zrzutów tabel
+--
+
+--
+-- Indexes for table `Category`
+--
+ALTER TABLE `Category`
+  ADD PRIMARY KEY (`id`);
```

Comment:
Deklaracje kluczy (głównych, unikalnych, zwykłych) możesz od razu przenieść do CREATE TABLE ... Kamil może Ci pokazać jak to jest zrobione w Shoperze: .sql/1.schema.sql